internal/pkg/manifest/lb_web_svc.go

```diff
@@ -183,6 +183,7 @@ func (s LoadBalancedWebService) ApplyEnv(envName string) (WorkloadManifest, erro
 // RoutingRule holds the path to route requests to the service.
 type RoutingRule struct {
 	Path                *string                 `yaml:"path"`
+	ProtocolVersion     *string                 `yaml:"protocolVersion"`
```

Comment:
What do you think of renaming the field like this? Since it's already under `http`.
```suggestion
	ProtocolVersion     *string                 `yaml:"version"`
```

_Side note: Copilot uses snake_case for field names, so protocol_version instead of protocolVersion_.

Reply:
OK. I will rename to `version`.